### ISSUE TITLE
Zooming in extends pagemark

### ISSUE THREAD

**Issue by gmertk** [burtonator/polar-bookshelf#630]
NOTE: Polar is supported by community donations.  Bug reports and features are prioritized for people who make a donation: https://opencollective.com/polar-bookshelf

**Describe the bug**
Say that we have a pagemark and the zoom level is 125%. 
Increasing the zoom level to 150% changes the pagemark, moves it further down.

**To Reproduce**
1. Open an article.
2. Create a pagemark. Note that where it is.
3. Zoom in. 
4. Confirm that pagemark is extended.

**Expected behavior**
The pagemark shouldn't change when the zoom level changes.

**Desktop (please complete the following information):**
 - macOS High Sierra 10.13.6 
 - Chrome Version 72.0.3626.109

**Comment by burtonator** [comment]
Yes... agreed.. it's definitely very annoying but I don't have a short term fix for this yet.
